test/smoke/framework/cli.go

```diff
@@ -71,7 +71,7 @@ func CliDeployExample(namespace, appName, contextDir, runtime string, native, pe
 		cmd = append(cmd, "--build-env", fmt.Sprintf("%s=-Ppersistence", mavenArgsAppendEnvVar))
 	}
 
-	// TODO setupBuildImageStreams(kogitoApp)
+	cmd = append(cmd, "--image-version", getEnvImageVersion())
```

Comment:
not sure if I understand this replacement.

Reply:
or maybe the TODO was outdated? :)

Reply:
TODO outdated since we don’t have a custom image stream anymore. We are now handling these